packages/mdc-drawer/README.md

```diff
@@ -437,55 +437,80 @@ document.body.addEventListener('MDCDrawer:closed', () => {
+`elementHasClass(element: Element, className: string) => boolean` | Returns true if the element contains the given class.
 `releaseFocus() => void` | Releases focus trap from root element which was set by `trapFocus` and restores focus to where it was prior to calling `trapFocus`.
+`removeClass(className: string) => void` | Removes a class from the root Element.
+`restoreFocus() => void` | Restores focus to element previously saved with 'saveFocus'.
+`saveFocus() => void` | Saves the focus of currently active element.
+`trapFocus() => void` | Traps focus on root element and focuses the active navigation element.
+
 
-### Foundations
+### MDCDismissibleDrawerFoundation
 
-#### `MDCDismissibleDrawerFoundation`
+#### Methods
 
-Method Signature | Description
+Signature | Description
 --- | ---
-`open() => void` | Opens the drawer from the closed state.
 `close() => void` | Closes the drawer from the open state.
+`handleKeydown(evt: KeyboardEvent) => void` | Keydown handler to close drawer when key is escape.
+`handleTransitionEnd(evt: TransitionEvent) => void` | Handles the `transitionend` event when the drawer finishes opening/closing.
+`isClosing() => boolean` | Returns true if the drawer is animating closed.
 `isOpen() => boolean` | Returns true if the drawer is in the open position.
 `isOpening() => boolean` | Returns true if the drawer is animating open.
-`isClosing() => boolean` | Returns true if the drawer is animating closed.
-`handleKeydown(evt: Event) => void` | Handles the `keydown` event.
-`handleTransitionEnd(evt: Event) => void` | Handles the `transitionend` event when the drawer finishes opening/closing.
+`open() => void` | Opens the drawer from the closed state.
+
 
-#### `MDCModalDrawerFoundation` (extends `MDCDismissibleDrawerFoundation`)
+### MDCModalDrawerFoundation
 
-Method Signature | Description
+#### Methods
+
+Signature | Description
 --- | ---
+`close() => void` | Closes the drawer from the open state.
+`handleKeydown(evt: KeyboardEvent) => void` | Keydown handler to close drawer when key is escape.
 `handleScrimClick() => void` | Handles click event on scrim.
+`handleTransitionEnd(evt: TransitionEvent) => void` | Handles the `transitionend` event when the drawer finishes opening/closing.
+`isClosing() => boolean` | Returns true if the drawer is animating closed.
+`isOpen() => boolean` | Returns true if the drawer is in the open position.
+`isOpening() => boolean` | Returns true if the drawer is animating open.
+`open() => void` | Opens the drawer from the closed state.
+
+
+
+<!-- docgen-tsdoc-replacer:end -->
+
+## Usage within Web Frameworks
+
+If you are using a JavaScript framework, such as React or Angular, you can create a Navigation Drawer for your framework. Depending on your needs, you can use the _Simple Approach: Wrapping MDC Web Vanilla Components_, or the _Advanced Approach: Using Foundations and Adapters_. Please follow the instructions [here](../../docs/integrating-into-frameworks.md).
```

Comment:
Can we move this to handlebar template and replace `...create a Navigation Drawer for your framework...` with `...create this component for your framework...` to not break the order.

This section comes after component API table and before adapter / foundation table.